pkg/storaged/dialog.js

```diff
@@ -304,8 +315,9 @@
             return (errors.length === 0)? vals : null;
         }
 
-        $dialog.on('change input', function () {
+        $dialog.on('change input', function (event) {
             update_visibility();
+            update_fields($(event.target).attr("data-field"));
```

Comment:
It is safe in the sense that we don't get an exception, but it would still be a bug.

It so happens that the current code only is interested in text inputs and checkboxes, and they both have a `data-field` attribute.  Most fields have this attribute on the element that triggers the event, but some do not, such as SizeSliders and the dropdowns.

So this is indeed a bug.  (Which I knew about but I still took the shortcut for 'performance'...)